index.html

```diff
@@ -65,53 +76,89 @@ <h2 class="mt3 pb1">Changes to DuckDuckHack</h2>
+                        and we are grateful to know that, as we continue to grow, more and more people are eagerly
+                        awaiting the next opportunity to get involved.</p>
 
-                    <p class="pv2">We’re not sure what the next community initiative will look like, or when it will be ready to share with you. However, we actively maintain several other open-source projects, such as our Android and iOS apps, browser extensions, and more, which you can you find on <a href="https://github.com/duckduckgo/">GitHub</a>.</p>
+                    <p class="pv2">We’re not sure what the next community initiative will look like, or when it will be
+                        ready to share with you. However, we actively maintain several other open-source projects, such
+                        as our Android and iOS apps, browser extensions, and more, which you can you find on <a
+                            href="https://github.com/duckduckgo/">GitHub</a>.</p>
 
-                    <p class="pv2">We’re very thankful to the many developers and Community Leaders who’ve volunteered their time over the years to DuckDuckHack and we hope to work with you all again soon!</p>
+                    <p class="pv2">We’re very thankful to the many developers and Community Leaders who’ve volunteered
+                        their time over the years to DuckDuckHack and we hope to work with you all again soon!</p>
 
                     <p class="pv2"><br><br>Sincerely,<br>The DuckDuckGo Staff</p>
                 </section>
-            </div>
-        </div>
 
-        <!-- footer -->
-        <div class="blk  blk--foot  blk--foot--nav ">
-            <div class="cw--c">
-                <div class="blk--foot__nav-col  third  thirty--screen-s">
-                    <h5 class="blk--foot__nav-head">DuckDuckGo</h5>
-                    <ul>
-                        <li><a href="https://duckduckgo.com/about">About</a></li>
-                        <li><a href="https://duckduckgo.com/bang">Bangs</a></li>
-                        <li><a href="https://duckduckgo.com/spread">Spread</a></li>
-                        <li><a href="https://duckduckgo.com/hiring">Careers</a></li>
-                        <li><a href="https://duck.co/help">Help</a></li>
-                    </ul>
-                </div>
-                <div class="blk--foot__nav-col  third  thirty--screen-s">
-                    <h5 class="blk--foot__nav-head">Privacy</h5>
-                    <ul>
-                        <li><a href="https://spreadprivacy.com/">Blog</a></li>
-                        <li><a href="https://donttrack.us/">DontTrack.us</a></li>
-                        <li><a href="https://duckduckgo.com/privacy">Policy</a></li>
-                    </ul>
-                </div>
-                <div class="blk--foot__nav-col  third  thirty--screen-s">
-                    <h5 class="blk--foot__nav-head">Keep in Touch</h5>
+                <hr>
+
+                <section>
```

Comment:
The below content is new. The changes above are just formatting changes, the text hasn't been modified.